### DISCUSSION TITLE
speed after recent upgrade

### DISCUSSION THREAD

**Discussion by akgroupllc** [inventree/InvenTree#8576] [General]
after the most recent update the speed seems to have took a dive. 
I am hosting on digital ocean and in some cases i have to reboot the server to get back in. 
Recently i have been working on customizing label templates and things just grinded to stop and needed another reboot.
Any suggestions?

**Comment by SchrodingersGat** [comment]
What version did you upgrade from, and to? 

You will need to provide specific details of what is "slow". The user interface? API calls? Initial page load? Report generation, etc. 

Also if you're able to provide some actual data on how long these slow operations are taking. With server logs etc if available. 

There's not much we can do without any of that information

**Reply by akgroupllc** [reply]
i am on  0.16.8 unfortunately i do not remember which version we had before but it wasn't super far behind maybe just one or two releases behind.
It's weird because it seems to be the user interface and it seems to be inconsistent, i will try to gather more data but for example today i was working on the label templates and things were working just fine but then eventually something happened and i began getting timeout errors.  thanks i will try to gather more data.
